content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -28,15 +60,9 @@ Let's pick one image from this list and verify its signature using
 the `cosign verify` command:
 
 ```shell
-COSIGN_EXPERIMENTAL=1 cosign verify registry.k8s.io/kube-apiserver-amd64:v1.24.0
+cosign verify registry.k8s.io/kube-apiserver-amd64:v1.26.0
```

Comment:
```suggestion
cosign verify registry.k8s.io/kube-apiserver-amd64:v{{< skew currentVersion >}}.0
```

Extra credit, very much not blocking any merge: look up the latest patch version too.